src/chronify/csv_io.py

```diff
@@ -5,15 +5,28 @@
 from duckdb import DuckDBPyRelation
 
 from chronify.models import CsvTableSchema, get_duckdb_type_from_sqlalchemy
+from chronify.time_configs import DatetimeRange
 
 
 def read_csv(path: Path | str, schema: CsvTableSchema, **kwargs: Any) -> DuckDBPyRelation:
     """Read a CSV file into a DuckDB relation."""
     if schema.column_dtypes:
-        dtypes = {x.name: get_duckdb_type_from_sqlalchemy(x.dtype) for x in schema.column_dtypes}
+        dtypes = {
+            x.name: get_duckdb_type_from_sqlalchemy(x.dtype).id for x in schema.column_dtypes
+        }
         rel = duckdb.read_csv(str(path), dtype=dtypes, **kwargs)
     else:
         rel = duckdb.read_csv(str(path), **kwargs)
 
-    expr = ",".join(rel.columns)
+    time_config = schema.time_config
+    exprs = []
+    for i, column in enumerate(rel.columns):
+        expr = column
+        if isinstance(time_config, DatetimeRange) and column == time_config.time_column:
+            time_type = rel.types[i]
+            if time_type == duckdb.typing.TIMESTAMP and time_config.start.tzinfo is not None:  # type: ignore
+                expr = f"timezone('{time_config.start.tzinfo.key}', {column}) AS {column}"  # type: ignore
+        exprs.append(expr)
```

Comment:
This was the main problem. Now we are applying time zones.

Reply:
It seems like we may not need any of the timezone enums we created in time.py since we can use tzinfo directly.

Reply:
That might be the case. We might need them if we accept user input definitions as command line arguments or in configuration files.